src/theft_rng-bak.c

```diff
@@ -0,0 +1,152 @@
+/* 
```

Comment:
This backup file probably shouldn't be checked in...

Reply:
Sorry! Fixed.